PowerShellTools/DebugEngine/ScriptDebugger.cs

```diff
@@ -440,6 +440,7 @@ public void Execute(ScriptProgramNode node)
                 if (node.IsFile)
                 {
                     commandLine = String.Format(DebugEngineConstants.ExecutionCommandFormat, node.FileName, node.Arguments);

```

Comment:
Was going off of what we do in OutputStringLine, but I am willing to acquiesce to your way.